MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1438,14 +1478,8 @@ private static SubtitleProfile GetExternalSubtitleProfile(MediaSourceInfo mediaS
             return null;
         }
 
-        private bool IsItemBitrateEligibleForDirectPlay(MediaSourceInfo item, long maxBitrate, PlayMethod playMethod)
+        private bool IsBitrateLimitExceeded(MediaSourceInfo item, long maxBitrate)
         {
-            // Don't restrict by bitrate if coming from an external domain
-            if (item.IsRemote)
-            {
-                return true;
-            }
-
```

Comment:
I'm not sure why this is relevant here.

Reply:
I'm only guessing, but if the stream is remote, the bitrate limit doesn't apply because the stream doesn't come from the server.

Reply:
Is the item proxied by the server or directly passed to the client without the server being involved? Because if the server proxies the stream the limit should imo be enforced too.